apps/storybook/src/TiledHeatmapMesh/checkerboard-api.ts

```diff
@@ -0,0 +1,53 @@
+import type { Size } from '@h5web/lib';
+import { getLayerSizes, TilesApi } from '@h5web/lib';
+import greenlet from 'greenlet';
+import { clamp } from 'lodash';
+import type { NdArray } from 'ndarray';
+import ndarray from 'ndarray';
+import { createFetchStore } from 'react-suspense-fetch';
+import type { Vector2 } from 'three';
+
+import type { TileParams } from './models';
+import { areTilesEqual } from './utils';
+
+const getCheckerboardArray = greenlet(
```

Comment:
The checkerboard tiles arrays are now computed in a worker like with the mandlebrot API. This gets rid of a lot of jank when playing with the `Checkerboard` story. (Note also the renaming of the word "checkboard" to "checkerboard" everywhere.)